reccmp/isledecomp/parser/util.py

```diff
@@ -20,7 +20,7 @@
 regularCommentRegex = re.compile(r"(//.*)")
 
 # Get string contents, ignore escape characters that might interfere
-doubleQuoteRegex = re.compile(r"(\"(?:[^\"\\]|\\.)*\")")
+doubleQuoteRegex = re.compile(r"(L)?(\"(?:[^\"\\]|\\.)*\")")
```

Comment:
I just realised that this regex can be simplified a bit by enclosing it in single quotes.

`r'(L)?("(?:[^"\\]|\\.)*")'` should be equivalent (assuming our linter does not complain, but even then I'd add an exception)